### PR TITLE
Fix every PHPStan problem and more

### PR DESCRIPTION
**Pull request by szepeviktor**
- removed redundant logic
- check source code
- tests too
- WordPress code too

### CODE REVIEW

File: grumphp.yml [hunk 1]
```diff
@@ -26,8 +26,7 @@ grumphp:
       allow_risky: true
       config: '.php-cs-fixer.dist.php'
     phplint: ~
-    phpstan:
-      ignore_patterns: ['/^(?!src)/']
+    phpstan: ~
```
Comment (szepeviktor) [inline grumphp.yml:29]: I do not know GrumPHP.
I'm in contact with every tool author and executing those tools directly.

Comment (carlalexander) [reply grumphp.yml:29]: Basically this said to ignore everything that isn't in the `src` directory. I don't usually run PHPStan against tests.

File: composer.json [hunk 2]
```diff
@@ -40,8 +40,10 @@
         "php-parallel-lint/php-parallel-lint": "^1.1",
         "phpro/grumphp": "^1.0",
         "phpstan/phpstan": "^1.0",
+        "phpstan/phpstan-phpunit": "^1.3",
         "phpunit/phpunit": "^9.3",
-        "sebastian/phpcpd": "^6.0.3"
+        "sebastian/phpcpd": "^6.0.3",
+        "szepeviktor/phpstan-wordpress": "^1.1"
```
Comment (carlalexander) [inline composer.json:46]: Why did we need the WordPress rules for the CLI?

**Comment by carlalexander** [comment]
Thanks and sorry for the delay! I was on vacation 😄  

I'm reviewing and it seems the fixes are all stylistic. Are you just looking to have PHPStan run correctly without GrumPHP?

File: composer.json (hunk 2 above)
Comment (szepeviktor) [reply composer.json:46]: We have an MU plugin for WordPress.
https://github.com/ymirapp/cli/blob/main/stubs/activate-ymir-plugin.php

**Comment by szepeviktor** [comment]
> Are you just looking to have PHPStan run correctly without GrumPHP?

Yes. I'm looking for people who need a big ✔️ from PHPStan.

**Pull request closed without merging**